resources/user_config.js

```diff
@@ -39,8 +39,25 @@ let UserConfig = {
 
       // The plugin auto-detects the language, so set this first.
       // If options files want to override it, they can for testing.
-      if (e.detail.language)
-        Options.Language = e.detail.language;
+      if (e.detail.parserLanguage) {
+        Options.ParserLanguage = e.detail.parserLanguage;
+        // Backward compatibility, everything "Language" should be changed to "ParserLanguage"
+        Options.Language = e.detail.parserLanguage;
```

Comment:
Could you add a section at the top with this code, that would then be overwritten by parserLanguage and systemLocale and displayLanguage?

```
if (e.detail.language) {
  Options.ParserLanguage = e.detail.language;
  Options.ShortLocale = e.detail.language;
  Options.DisplayLanguage = e.detail.language;
}
```

Mostly so that folks who are using the latest on github won't break when this lands? I don't normally care that much about breaking people on plugin updates, but this seems like an easy fix.